drivers/boston_project_scorecard/app/models/boston_project_scorecard/report.rb

```diff
@@ -155,7 +155,7 @@ def project_name
     end
 
     def title
-      _('MA-500 Boston Continuum of Care FY21 Renewal Project Scoring Tool')
+      _('MA-500 Boston Continuum of Care FY2022 Renewal Project Scoring Tool')
```

Comment:
can you run `rake gettext:sync`?  We should be doing this any time we add/change translations.